src/transformers/models/vision_text_dual_encoder/configuration_vision_text_dual_encoder.py

```diff
@@ -0,0 +1,129 @@
+# coding=utf-8
+# Copyright The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" VisionTextDualEncoder model configuration """
+
+import copy
+
+from ...configuration_utils import PretrainedConfig
+from ...utils import logging
+from ..auto.configuration_auto import AutoConfig
+from ..clip.configuration_clip import CLIPVisionConfig
+
+
+logger = logging.get_logger(__name__)
+
+
+class VisionTextDualEncoderConfig(PretrainedConfig):
+    r"""
+    :class:`VisionTextDualEncoderConfig` is the configuration class to store the configuration of a
+    :class:`~VisionTextDualEncoderModel`. It is used to instantiate VisionTextDualEncoderModel model according to the
```

Comment:
The syntax is :class:`~transformers.ClassName`. `VisionTextDualEncoderModel` should also have its `:class:`.